Makefile.PL

```diff
@@ -250,7 +250,9 @@ WriteMakefile(
 		   'File::Temp'  => 0,
 		   'HTTP::Tiny'  => 0,
 		   'POSIX'       => 0,
-		   'File::Spec'  => 0
+		   'File::Spec'  => 0,
+		   'Test::More'  => 0,
+		   'Test::Exception' => 0
```

Comment:
These need to be instead a `TEST_REQUIRES`

Reply:
These have been moved out to TEST_REQUIRES in d412e49

Reply:
Great